addons/website_event/views/event_templates.xml

```diff
@@ -203,9 +203,6 @@
                         <div t-if="event.organizer_id" class="mb-3">
                             <small class="text-muted text-truncate">Organizer: <span t-field="event.organizer_id" itemprop="organizer"/></small>
                         </div>
-                        <!-- Location -->
-                        <div t-if="event.is_online">Online</div>
-                        <div t-else="" itemprop="location" t-field="event.address_id" t-options="{'widget': 'contact', 'fields': ['city'], 'no_marker': 'true'}"/>
```

Comment:
We should not remove the address `t-if="not event.is_online"`

Reply:
You did not answer its comment and now he is sad.

Reply:
zizisse because the location has been put below.